### PR TITLE
add changelog for 6.0.0

### PR DESCRIPTION
**Pull request by mmoll**
[ci skip]

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+## 6.0.0
+* Drop Puppet 3 support
+* New or changed parameters:
+    * Add `$autosignfile` for 1.16+ puppetca proxies and `$use_autosignfile` to be able to still use this module
+      with older proxy versions.
+* New or changed parameters on smart proxy plugin classes:
+    * Add `$ssl_disabled_ciphers` to the foreman_proxy::plugin::dynflow class.
+    * Add `$collect_status` to the foreman_proxy::plugin::monitoring class.
+* Other changes and fixes:
+    * Set foreman_smartproxy features for built-in and plugin modules.
```
Comment (ekohl) [inline CHANGELOG.md]: I'd add some description of why this is a great thing to have.

Comment (mmoll) [inline CHANGELOG.md]: I added a sentence there, but would be grateful for better wordings, if wanted :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+## 6.0.0
+* Drop Puppet 3 support
+* New or changed parameters:
+    * Add `$autosignfile` for 1.16+ puppetca proxies and `$use_autosignfile` to be able to still use this module
+      with older proxy versions.
+* New or changed parameters on smart proxy plugin classes:
+    * Add `$ssl_disabled_ciphers` to the foreman_proxy::plugin::dynflow class.
+    * Add `$collect_status` to the foreman_proxy::plugin::monitoring class.
+* Other changes and fixes:
+    * Set foreman_smartproxy features for built-in and plugin modules. This can be used to install and enable
```
Comment (ekohl) [inline CHANGELOG.md]: I'd propose:

Set foreman_smartproxy features for built-in and plugin modules. This verifies the proxy has correctly registered with all the desired features.

**Comment by mmoll** [comment]
should be all done now :bowtie:

**Pull request merged**